### PR TITLE
Add support to environment variables (Advanced options)

### PR DESCRIPTION
**Pull request by atomsfat**
Fix https://issues.jenkins-ci.org/browse/JENKINS-40268

New advanced options:
image

### CODE REVIEW

**Comment by atomsfat** [comment]
@anallely please have a look on this

File: README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ doesn't require `ruby-runtime` to be installed.
 
 ## Usage
 - Please follow this [steps](https://wiki.jenkins-ci.org/display/JENKINS/Nvm+Wrapper+Plugin)
-                                                                           
+
 ## Build
-- It was builded using gradle 2.3 and Java 1.8
+- It was built using gradle 2.3 and Java 1.8
```
Comment (anallely) [inline README.md:12]: +1

File: src/main/java/org/jenkinsci/plugins/nvm/NvmWrapperUtil.java [hunk 2]
```diff
@@ -83,57 +97,38 @@ private String getExport(ArgumentListBuilder args, String destFile) throws IOExc
       .stderr(listener.getLogger()).join();
 
     if (statusCode != 0) {
-      new AbortException("Failed to fork bash ");
+      throw new AbortException("Failed to fork bash ");
```
Comment (anallely) [inline src/main/java/org/jenkinsci/plugins/nvm/NvmWrapperUtil.java:100]: XD

**Comment by anallely** [comment]
+1

**Pull request merged**